kafka-connect-azure/src/main/java/io/confluent/connect/azblob/format/parquet/ParquetRecordWriterProvider.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.confluent.connect.azblob.format.parquet;
+
+import io.confluent.connect.avro.AvroData;
+import io.confluent.connect.azblob.AzBlobSinkConnectorConfig;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.errors.ConnectException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.parquet.avro.AvroParquetWriter;
+import org.apache.parquet.hadoop.ParquetWriter;
```

Comment:
I didn't think this project supported Parquet. See issue #26.

Reply:
I have read that issue. After reading it, it was not completely clear to me what the issues with AvroParquetWriter are. 
We need Parquet support so I decided to give it a shot.

Generally, issue #26 talks about two type of issues
1. Implementation issues ("requires an HDFS configuration") -- I actually found it very easy to implement, the AvroParquetWriter has a builder and it just requires an HDFS path which can be easily constructed from the filename.
2. Runtime issues with delivery -- It works for us, though we only tested in standalone mode. I'm not sure which particular issues we need to watch out for. Also, some issues may be specific for S3 and not Azure blob, since these services are different.

What would you propose to do with Parquet?
- remove support
- do more testing
- something else?

Reply:
I think the commentary in that comment was referring to needing these

```
import org.apache.hadoop.conf.Configuration;
import org.apache.hadoop.fs.Path;
```

And the constructor for `AvroParquetWriter` that only uses a Path (which is an HDFS Path object, as you mention), is deprecated in favor of the one for the `Configuration` object. 

I really don't know the specifics of the delivery guarantees. That needs to be clarified, but it makes sense to me that HDFS dependencies shouldn't be needed, even if the cloud storage providers may offer HDFS compatibility. Parquet is tightly coupled to the HDFS Configuration, though, so there is no way around that to support the filetype, IMO.